content/tagging/using_tags.md

```diff
@@ -12,105 +12,271 @@ further_reading:
   text: Learn how to assign tags
 ---
 
-After you have assigned tags at the host and [integration][1] level, you can start using them to filter and group in interesting ways. There are several places you can use tags:
+After [assigning tags][1], you can start using them to filter and group in interesting ways.
```

Comment:
no `you can` and `and group in interesting ways.` => Too vague 

r/`After [assigning tags][1], you can start using them to filter and group in interesting ways.`/`After [assigning tags to your data][1], start using them to filter and group your Data in your Datadog platform.`